lib/tests/delta_generator_test_case.py

```diff
@@ -63,8 +57,6 @@ def setUp(self):
         )
         add_script_run_ctx(threading.current_thread(), self.script_run_ctx)
 
-        self.app_session = FakeAppSession()
```

Comment:
This was the primary warnings culprit! It turns out it's not actually used anywhere.